packages/aws-arch/ASSETS.md

```diff
@@ -0,0 +1,2824 @@
+<!-- AUTO-GENERATED - DO NOT EDIT -->
+  # AWS Architecture Icons
+
+  This package generated normalized asset dictionary from [AWS Architecture Icons](https://aws.amazon.com/architecture/icons/).
+
+  Asset paths follow `<category>/<service>/<resource>[.<theme>].<ext>` structure.
+
+  > eg: storage/s3/bucket.png, storage/s3/bucket.dark.png=
+
+  **Category** icons follow `<category>/category_icon.<ext>` structure.
+
+  > eg: storage/category_icon.png
+
+  **Service** icons follow `<category>/<service>/service_icon.<ext>` structure.
+
+  > eg: storage/s3/service_icon.png
+
+  The _default theme_ is **light**.
+
+---
+
+## Available assets
```

Comment:
Since removing the assets directory from source code (now only part of lib artifact), this file gives users visibility into the available assets within this package. Otherwise would have to install the package and looking into node_modules, or clone the repo and generate assets to see what is available.

So I believe it is quite useful for end users.